safe-global/web-core
# fix: Dismiss update safe notification for 90 days

## What it solves

Resolves #1807 

## How this PR fixes it

- Stores a flag in localStorage with an expiry date of 90 days when the user closes the update safe notification

## How to test it

1. Open a Safe with an older version
2. Observe the update safe notification
3. Dismiss the notification by either pressing "x" in the corner or the link
4. Observe a new localStorage entry in the format of `SAFE_v2__dismissUpdateSafe_<chainId>:<safeAddress>` that contains a timestamp for 90 days in the future
5. Manually change the timestamp to one in the past e.g. by using `Date.now()`
6. Reload the page
7. Observe the localStorage entry is deleted
8. Observe the update safe notification showing up

## Checklist
